### DISCUSSION TITLE
Discussion of issues raised in using the FedEx API

### DISCUSSION THREAD

**Discussion by dev-miro26** [community/community#128388] [Programming Help]
### Select Topic Area

General

### Body

I worked on a shopping and delivery site made with Laravel + Nuxt.js.
However, we had a lot of trouble using the FedEx API and were unable to implement the API later.
If you have experience using the FedEx API, please leave a message.(shipping fee and address validation etc....)
Thank you.

**Comment by ebndev** [comment]
Hi @dev-miro26, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our Programming Help category, as this topic might be more relevant there and hopefully might help someone give you a nudge in the right direction.
